components/library/librsvg/test/results-all.master

```diff
@@ -1,23 +1 @@
-test result: ok. 265 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 5 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 13 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 3 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 14 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 21 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 24 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 4 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 11 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 749 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 38 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 9 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 6 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 8 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 5 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 23 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 100 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 16 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
-test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; 
+test result: FAILED. 99 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out;
```

Comment:
What is causing the test failure?  I'd prefer to see all tests passed to avoid surprises.

Reply:
`rsvg_convert`?  That sounds suspicious (maybe related to gimp?).  Could you please try to fix it?